zerver/lib/actions.py

```diff
@@ -7756,7 +7753,7 @@ def do_update_user_custom_profile_data_if_changed(
                 # to a string for the comparison in this if.
                 continue
 
-            field_value.value = custom_profile_field["value"]
+            field_value.value = str(custom_profile_field["value"])
             if field_value.field.is_renderable():
                 field_value.rendered_value = render_stream_description(
                     str(custom_profile_field["value"])
```

Comment:
We can probably remove this `str()` conversion as well, right?

Reply:
I see this happens in a later commit, resolving.

Reply:
That was removed in the last commit (in favor of using `custom_profile_field_value_string` - because `render_stream_description` wants a `str` as argument)